Graphs/TreeGraph/index.js

```diff
@@ -228,6 +231,67 @@ const TreeGraph = (props) => {
         }
     },  [props.params, props.data, nodeElement, pageNo, kids]);
 
+    const downloadTreeGraph = async (name) => {
+        count++;
+        await htmlToImage.toJpeg(document.getElementsByClassName('tree-graph').item(0), { quality: 0.95 })
+            .then(function (dataUrl) {
+                let link = document.createElement('a');
+                link.download = `${name}-snapshot-${count}.jpeg`;
+                link.href = dataUrl;
+                link.click();
+            });
+            await new Promise(resolve => setTimeout(resolve, 2000));
+    }
+
+    const onDownloadClick = async () => {
```

Comment:
could we use onClick to with useRef with toPng rather than use all jQuery to look up the internals of the node for this?